crates/m3/src/builder/table.rs

```diff
@@ -839,4 +888,40 @@ mod tests {
 		let tb_ns_2 = tb_ns_1.with_namespace("ns2");
 		assert_eq!(tb_ns_2.namespaced_name("column"), "ns1::ns2::column");
 	}
+
+	#[test]
+	fn test_read_method() {
+		let mut cs = ConstraintSystem::<B128>::new();
+		let lookup_chan = cs.add_channel("lookup");
+		let mut read_table = cs.add_table("stacked_reads");
+		const LOG_STACKING_FACTOR: usize = 3;
+		const LOG_READ_TABLE_SIZE: usize = 3;
+		let read_col =
+			read_table.add_constant("constant_reads", [B8::new(3); 1 << LOG_STACKING_FACTOR]);
+		read_table.require_fixed_size(LOG_READ_TABLE_SIZE);
+
+		read_table.read(lookup_chan, [read_col]);
+		drop(read_table);
+		let mut write_table = cs.add_table("packed_write");
+		write_table.require_fixed_size(LOG_READ_TABLE_SIZE);
+		let write_col = write_table.add_constant("constant_writes", [B8::new(3)]);
+		write_table.push_with_opts(
+			lookup_chan,
+			[write_col],
+			FlushOpts {
+				multiplicity: 1 << LOG_STACKING_FACTOR as u32,
+				selectors: vec![],
+			},
+		);
+
+		let alloc = Bump::new();
+		let mut witness: WitnessIndex<PackedType<OptimalUnderlier, B128>> =
+			WitnessIndex::new(&cs, &alloc);
+		witness.fill_constant_cols().unwrap();
```

Comment:
Why is it here and does it do anything? I am asking because it seems that this should not do anything with tables that haven't been initialized before.